(Edit)Added SortCommandParserTest.Updated UserGuide

DarrenCsAcc opened the pull request:
sort command parser test added

DarrenCsAcc committed:
Added SortCommandParserTest

DarrenCsAcc committed:
Merge branch 'AY2223S1-CS2103T-W10-3:master' into master

DarrenCsAcc committed:
Updated command table in user guide
Added remove as a feature in the user guide
Will update the rest after this

DarrenCsAcc commented:
I added remove in the user guide as well as updated the table of the user guide

DarrenCsAcc committed:
added sort in the user guide
edited filter

DarrenCsAcc commented:
Added sort in user guide. Edited filter as discussed(filter [as/APPLICATIONSTATUS]).

ShummyOwnzYou commented on docs/UserGuide.md:
```diff
@@ -164,6 +164,30 @@ Exits TrackAScholar.
+*  `remove accepted` removes all entries with ACCEPTED as an application status from TrackAScholar.
+*  `remove rejected` removes all entries with REJECTED as an application status from TrackAScholar.
+
+### Removing applicants by status : `sort`
```
Header name doesn't correspond with command name.

DarrenCsAcc replied:
Alright i will resolve it and let you know when i am done!

ShummyOwnzYou commented on src/test/java/seedu/address/logic/parser/SortCommandParserTest.java:
```diff
@@ -0,0 +1,55 @@
+import seedu.address.logic.commands.SortCommand;
+import seedu.address.model.applicant.Applicant;
+
+public class SortCommandParserTest {
```
Need to add test case for sort by status, newly added functionality.

DarrenCsAcc replied:
Same for this!

ShummyOwnzYou reviewed (commented):
LGTM, other than the 2 comments that I've mentioned.

DarrenCsAcc committed:
Fixed mistakes and made necessary changes as proposed by the team

DarrenCsAcc committed:
Merge branch 'AY2223S1-CS2103T-W10-3:master' into master

DarrenCsAcc committed:
Update SortCommandParser Test

DarrenCsAcc committed:
Update the
developer guide - Added Find.Need to update it.
User guide
and sortcommand parser test

benjytan45678 commented:
LGTM, the test cases can be further improved in v1.4
